cfme/intelligence/reports/__init__.py

```diff
@@ -30,9 +30,13 @@ def is_displayed(self):
         return self.in_intel_reports and self.configuration.is_displayed
 
     def correct_tree_item_selected(self):
-        # The bug is currently targeted only to 5.10, but we have the
-        # problem on 5.11 as well, so we probably need to use
-        # forced_streams.
+        # The bug is currently deferred, so normal BZ(*).blocks doesn't block.
+        # It is targeted only to 5.10, but we have the problem on 5.11 as well,
+        # so use of forced_streams is needed.
+        #
+        # This workaround is needed for the test_dashboard_crud and perhaps
+        # some other tests.
+        #
```

Comment:
We can remove these comments now too!

Reply:
Thanks. I removed the first paragraphs. The second one can be useful in the future for tracking down which tests are affected.